platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformComponentBuilder.java

```diff
@@ -534,4 +537,35 @@ public ConsensusEngine buildConsensusEngine() {
         }
         return consensusEngine;
     }
+
+    /**
+     * Provide a PCEs sequencer in place of the platform's default PCEs sequencer.
```

Comment:
Is this capitalization pattern intentional? `PCEs`

Reply:
Unintentional. Fixed.